Sources/UILocalNotification+Sweet.swift

```diff
@@ -5,7 +5,15 @@
         static let idKey = "idKey"
         static let fireDateKey = "fireDateKey"
 
-        public static func create(_ id: String, fireDate: Date, soundName: String? = nil, message: String, actionTitle: String? = nil) {
+        /// Schedules a local notification.
+        ///
+        /// - Parameters:
+        ///   - id: The id used to represent a local notification in your app.
+        ///   - fireDate: The date when the notification will trigger
+        ///   - soundName: The name of the sound to be used when the notification is received.
+        ///   - message: The message that the app will use for your local notification.
+        ///   - actionTitle: The title of the action to be displayed by your notification, it's optional, though, since Apple provides a good default for this.
+        public static func schedule(using id: String, fireDate: Date, soundName: String? = nil, message: String, actionTitle: String? = nil) {
```

Comment:
Hmmm its a bit difficult i guess. I would want it to be called 'id' so you know for sure that it's not something the user is going to be bothered with.